ui/src/members/components/SelectUsers.tsx

```diff
@@ -15,22 +16,39 @@ interface Props {
 
 export default class SelectUsers extends PureComponent<Props> {
   public render() {
-    const {users} = this.props
+    return <>{this.dropdownUsers}</>
```

Comment:
I don't think you need a fragment here and can just combine `render` and `dropdownUsers()`